Sources/web3swift/Utils/Extensions/String+Extension.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  String+Extension.swift
+//
+//
+//  Created by JeneaVranceanu on 17.10.2023.
+//
+
+import Foundation
+
+extension String {
+
+    func asJsonDictionary() throws -> [String: AnyObject]? {
+        guard let data = data(using: .utf8) else { return nil }
+        return try data.asJsonDictionary()
+    }
+}
```

Comment:
We're still haven't added SwiftLint ruleset here, and I wonder would we ever do this, but I believe that such a set that we're having in a stash contains the rule about keeping one empty line at the very end of each file.